app/reducers/builderReducer.ts

```diff
@@ -687,12 +687,31 @@ function _builderReducer(
     };
   }
   if (isAddEnumerationsAction(action)) {
+    // Deep copy clues
     const clues = { ...state.clues };
-    for (const [answer, answerClues] of Object.entries(clues)) {
-      clues[answer] = answerClues.map((clue) =>
-        parseClueEnumeration(clue) == null ? `${clue} (${answer.length})` : clue
-      );
+    for (const answer in clues) {
+      const answerClues = clues[answer];
+      if (answerClues != null) {
+        clues[answer] = answerClues.slice();
+      }
     }
+    // Modify only clues that have entries
+    forEachCluedEntry(
```

Comment:
This looks sketchier now, but I think should be safer in that it should only modify clues with entires. Still, ts doesn't make array access easy. Suggestions welcome on how to clean this up!

Reply:
Yeah it's a bit convoluted but I can't think of any cleanups that maintain the invariants here and I think it's worthwhile to only modify clues that have entries. So seems OK!